home-tasks/src/test/java/com/epam/tc/hw2/utils/SeleniumAbstractCore.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.tc.hw2.utils;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeSuite;
+
+public abstract class SeleniumAbstractCore {
+
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+
+    @BeforeSuite
+    public void beforeSuite() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeClass
+    public void setUp() {
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        driver.manage().window().maximize();
```

Comment:
I suggest to start browser already maximized. Please see second option in article https://unadkatjash.medium.com/maximizing-chrome-window-in-selenium-webdriver-using-java-73e20360f2d3